runtime/polkadot/src/lib.rs

```diff
@@ -337,6 +337,42 @@ impl pallet_session::historical::Config for Runtime {
 	type FullIdentificationOf = pallet_staking::ExposureOf<Runtime>;
 }
 
+parameter_types! {
+	// no signed phase for now, just unsigned.
+	pub const SignedPhase: u32 = 0;
+	// NOTE: length of unsigned phase is, for now, different than `ElectionLookahead` to make sure
+	// that we won't run OCW threads at the same time with staking. Once staking's election is out,
+	// replace with `/ 4`.
+	pub const UnsignedPhase: u32 = EPOCH_DURATION_IN_BLOCKS / 8;
```

Comment:
polkadot staking 
```suggestion
	pub const UnsignedPhase: u32 = EPOCH_DURATION_IN_BLOCKS / 32;
```
or if it works:
```
	pub const UnsignedPhase: u32 = ElectionLookAhead::get() / 2;
```

Reply:
yeah setting it to `ElectionLookAhead  / 2 makes more sense`